### ISSUE TITLE
Formatting rendering error when markup symbols are used in quote

### ISSUE THREAD

**Issue by Avimitin** [nanxiaobei/hugo-paper#179]
## Current rendering

<img src="https://github.com/nanxiaobei/hugo-paper/assets/30021675/75588913-3047-43c5-86da-9ab61efde3ed" width="50%"/>

## The content

```markdown
> ### Header in quote
>
> ```rust
> println!("Hi, I am a nested code block");
> ```
>
> Some other caption...
```

## Desired rendering

> ### Header in quote
>
> ```rust
> println!("Hi, I am a nested code block");
> ```
>
> Some other caption...

Using markup in quotes is important to me because I need it to briefly describe some of the topic. But the current implementation doesn't handle this case. It would be great if you could fix this.

**Comment by nanxiaobei** [comment]
I don't think this theme deals with rendering.

**Comment by Avimitin** [comment]
There are some misunderstanding. What I mean is that the code block occupying the left border of the quote, which can be fixed in css by adjusting the margin.

**Comment by nanxiaobei** [comment]
ok I got it, will fix it

**Comment by nanxiaobei** [comment]
fixed in `6.21.0`.

**Comment by Avimitin** [comment]
👍 thanks
